plugins/woocommerce/tests/e2e-pw/utils/wordpress.js

```diff
@@ -0,0 +1,60 @@
+const axios = require( 'axios' ).default;
+
+const getPreviousTwoVersions = async () => {
+	const response = await axios
+		.get( 'https://api.wordpress.org/core/version-check/1.7/' )
```

Comment:
Agree with using the `stable-check` endpoint. Fixed in 94552a67e4215b6876a42ee5549bc1b648c5051e.

We support up to the last 2 minor versions of WordPress, though, as noted [here](https://woocommerce.com/document/frequently-asked-questions/#wcvers). So getting those 2 previous versions isn't as simple as `versions[versions.indexOf(latest) - 1]`, unfortunately. Now that we're at WP 6.1.1, we will need to get `6.0.3` and `5.9.5` as the L-1 and L-2 versions, skipping `6.1.0`.

I updated the [loop](https://github.com/woocommerce/woocommerce/blob/94552a67e4215b6876a42ee5549bc1b648c5051e/plugins/woocommerce/tests/e2e-pw/utils/wordpress.js#L23-L39) for extracting those 2 versions so that it's simpler and much more straightforward than the previous one 🤞

Reply:
The new code for extracting L-1 and L-2 seems to work fine after re-running it just now 😌 
<img width="276" alt="jnO7C2CxjB" src="https://user-images.githubusercontent.com/4509348/215978985-607dcdf4-19b9-422c-9589-59f3ce151736.png">

Reply:
> We support up to the last 2 minor versions of WordPress, though

Ahhh I missed the "minor" part, I thought it was just the 2 latest versions in general.

> The new code for extracting L-1 and L-2 seems to work fine after re-running it just now

Awesome!! Great job 🙌